### PR TITLE
[DEV-852] Menu item should be active if subpath is included in path

### PR DESCRIPTION
**Pull request by MikeAtUqido**
#### List of Changes

* The product header menu item should be active even when the subpath is included in the path

#### Motivation and Context

The menu item lost the active prop when navigating in a subpath

#### How Has This Been Tested?

Manually

#### Screenshots (if appropriate):

https://github.com/pagopa/developer-portal/assets/146933303/e79daa25-6fd7-41ac-9016-2306d057dcb6

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/helpers/productHeader.helper.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function productToMenuItems(
       return {
         label: subpath.name,
         href: subpath.path,
-        active: path === subpath.path,
+        active: path.includes(subpath.path),
```
Comment (kin0992) [inline apps/nextjs-website/src/helpers/productHeader.helper.ts:16]: Could you add a unit test that catches this issue?

If we want to be more sure to not having troubles with paths I'd use the `startsWith` method, what do you think?
The corner case is:
`path` is `/a/b/c-d/e` and `subpath.path` is `/b/c-d`

**Review by MarcoPonchia — APPROVED** [review]
Everything is ok :+1:  Great that you also add some tests :muscle:

**Review by kin0992 — APPROVED** [review]
💯
